health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -501,7 +607,7 @@
 
 function enrichRootProjectId(requestBody: any) {
     var rootBoundary;
-    for (const boundary of requestBody?.CampaignDetails?.boundaries) {
+    for (const boundary of requestBody?.boundariesCombined) {
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)